src/webfaf/static/js/custom.js

```diff
@@ -172,6 +172,43 @@ $(document).ready(function() {
       $('#advanced-filters').removeClass('hide');
       $(this).addClass('hide');
     });
+
+    const observer = new ResizeObserver(entries => {
+      for (let entry of entries) {
+        const container = entry.target.querySelector('.crash-fn');
+        const expander = entry.target.querySelector('.expander');
+        const expanded = $(container).hasClass('expanded');
+        if (expanded && (container.scrollHeight == expander.scrollHeight)) {
+          container.classList.remove('expanded');
+
+          return;
+        }
+        const showExpander = !expanded && (container.scrollHeight > container.clientHeight);
+        const showCollapser = expanded;
+        if (showExpander) {
+          expander.innerHTML = '[+]';
```

Comment:
Suggest using [show more] and [show less] instead. The [+] and [-] remind me of forums with Reddit-like layout where it's used to expand/collapse a subtree of posts/messages and it looks a little odd centered at the bottom like this. But maybe it's just me.

Reply:
Looking for analogies here is a bit pointless, but I can make it textual, sure. Should be fine, given that we don’t localize strings.